src/index.html

```diff
@@ -11,6 +11,48 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+    action="#"
+    method="post"
+    data-qa="big"
+  >
+      <label>
+        <input
+        data-qa="keypress"
+        type="text"
+        name="search-big"
+        class="search search-big"
+        placeholder="Try &quot;Los Angeles&quot;"
+    >
+      </label>
+      <img
+        src="images/Search.svg"
+        alt="Search icon"
+        class="search-icon big-icon"
+      >
+      </label>
+    </form>
+
+    <form
+    action="#"
+    method="post"
+    data-qa="small"
+    >
+      <label>
+        <input
+        type="text"
+        name="search-small"
+        class="search search-small"
+        placeholder="Try &quot;Los Angeles&quot;"
+    >
+      </label>
+      <label>
+        <img
+        class="search-icon small-icon"
+        src="images/Search.svg"
+        alt="Search image"
+      >
+      </label>
```

Comment:
```suggestion
      <label for="">
        <input
          type="text"
          id=""
          name="search-small"
          class="search search-small"
          placeholder="Try &quot;Los Angeles&quot;"
        >
        <img
        class="search-icon small-icon"
        src="images/Search.svg"
        alt="Search image"
      >
      </label>
```
you should connect the label, the input and the picture